jsapp/js/components/processing/singleProcessingSubmissionDetails.tsx

```diff
@@ -154,6 +154,6 @@ export default class SingleProcessingSubmissionDetails extends React.Component<S
   }
 
   render() {
-    return [this.renderMedia(), this.renderDataList()];
+    return [this.renderMedia()];
```

Comment:
~~I think this file needs some cleanup :) AFAIU `renderDataList` can be removed? and this will only render media?~~ Do we delete this file?